pkg/x509.go

```diff
@@ -59,15 +60,22 @@ const TLSCertificateValidityInDays = 365
 // SigningCertificateValidityInDays holds the number of days issued signing certificates are valid
 const SigningCertificateValidityInDays = 365
 
-// Certificate validity in days according to Nuts RFC003
+// RFC003ValidityInDays is the number of days a certificate is valid according to Nuts RFC003
 const RFC003ValidityInDays = 4
 
-// RFC003CertificateProfile is a x509.CertificateProfile according to RFC003: signing the JWT bearer token
-var RFC003CertificateProfile = CertificateProfile{
+// RFC003SigningCertificateProfile is a x509.CertificateProfile according to RFC003: signing the JWT bearer token
+var RFC003SigningCertificateProfile = CertificateProfile{
 	KeyUsage:     x509.KeyUsageDigitalSignature | x509.KeyUsageContentCommitment,
 	NumDaysValid: RFC003ValidityInDays,
 }
 
+// RFC003TLSCertificateProfile is a x509.CertificateProfile according to RFC003/RFC008: setting up the TLS connection between nodes
+var RFC003TLSCertificateProfile = CertificateProfile{
+	KeyUsage:     x509.KeyUsageDigitalSignature | x509.KeyUsageKeyEncipherment | x509.KeyUsageKeyAgreement,
+	ExtKeyUsage:  []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
```

Comment:
The TLS client certificate now also has the server auth usage. Is that intended?

Reply:
copy paste error